app/models/internal_post.rb

```diff
@@ -0,0 +1,27 @@
+class InternalPost < Post
+	belongs_to :user
+	validates_presence_of :user
```

Comment:
Tab detected.<br>Prefer the new style validations `validates :column, presence: value` over `validates_presence_of`.